src/OrchardCore/OrchardCore.Data.YesSql/SqliteHelper.cs

```diff
@@ -6,10 +6,10 @@ namespace OrchardCore.Data;
 
 public static class SqliteHelper
 {
-    public static string GetConnectionString(SqliteOptions sqliteOptions, ShellOptions shellOptions, string shellName, string databaseName = "")
-        => GetConnectionString(sqliteOptions, GetDatabaseFolder(shellOptions, shellName), databaseName);
+    public static string GetConnectionString(SqliteOptions sqliteOptions, ShellOptions shellOptions, string shellName, string databaseName = "", bool createFile = false)
```

Comment:
Why this not done in YesSQL?

Reply:
Because the only thing YesSQL does is take a SQLite connection string. It doesn't know anything about an OC shell context.

Reply:
I knew, what I mean why YesSQL doesn't create a database from a supplied path

Reply:
We would need to assume that the connection string is always not using the ReadWriteCreate mode from YesSQL. It's not YesSQL that creates the SQLite database it is the SQLite driver. I don't think we want to change the connection string from YesSQL to prevent creating the file. It is in OC that we should use a different connection string from different context. We should not use the ReadWriteCreate mode all the time as it adds an extra check in the SQLite driver ...
To me, that makes sense to always use the ReadWrite mode. Then, we can't prevent someone from using the ReadWriteCreate mode either ...